exasol/analytics/query_handler/query/result/udf_query_result.py

```diff
@@ -51,7 +51,7 @@ def rowcount(self) -> int:
 
     def fetch_as_dataframe(
         self, num_rows: Union[str, int], start_col: int = 0
-    ) -> "pandas.DataFrame":
+    ) -> "pandas.DataFrame":  # type: ignore
```

Comment:
Would 
```suggestion
    ) -> Optional[pandas.DataFrame]: 
```
work here?

Reply:
Yes, but requires importing pandas. I was not sure whether the string syntax was intentional, here?

Reply:
Importing pandas can take several seconds. 
See next push, replacing this with an explicit forward reference.

Reply:
I still get errors. See also https://github.com/pydantic/pydantic/issues/234
As far as I can see, either mypy complains or pytest fails.
I don't see a way to use `Optional` together with a type, that is not imported, yet.